macos/FluentUITestApp/Assets.xcassets/ic_fluent_chevron_down_12_regular.imageset/Contents.json

```diff
@@ -0,0 +1,15 @@
+{
+  "images": [
+    {
+      "filename": "ic_fluent_chevron_down_12_regular.svg",
+      "idiom": "universal"
+    }
+  ],
+  "info": {
+    "author": "xcode",
+    "version": 1
+  },
+  "properties": {
+    "template-rendering-intent": "template"
+  }
+}
```

Comment:
Nit: This shows there's no blank line at the end of this file and there should be.

Reply:
This is copied over from FluentIcon library where there's no blank line at the end.